src/main/java/edu/ucsb/cs156/courses/controllers/PersonalSchedulesController.java

```diff
@@ -88,6 +88,11 @@ public PersonalSchedule postSchedule(
         CurrentUser currentUser = getCurrentUser();
         log.info("currentUser={}", currentUser);
 
+        // Checks length of name parameter (length should be 15 chars or less, nonzero)
+        if (name.length() > 15) {
```

Comment:
When testing this code, we can pass the mutation tests that change the boundary value by adding two tests: one that should accept with `name.length() == 15` (the maximum allowed length), and one that should reject because `name.length() == 16` (the minimum disallowed length).

Reply:
Yes... you do need tests... :-)